File: 09366.py
for i in range(int(input())):
    angle = sorted(list(map(int, input().split())))

    print("Case #%d:" % (i + 1), end=' ')
    if angle[2] >= angle[0] + angle[1]:
        print("invalid!")
    elif angle[0] == angle[1] and angle[0] == angle[2]:
        print("equilateral")
    elif angle[0] == angle[1] or angle[1] == angle[2]:
        print("isosceles")
    else:
        print("scalene")

File: 09366.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main() {
	int t; cin >> t;

	int angle[3];
	for (int i = 0; i < t; i++) {
		cin >> angle[0] >> angle[1] >> angle[2];
		sort(angle, angle + 3);

		cout << "Case #" << i + 1 << ": ";
		if (angle[2] >= angle[0] + angle[1])
			cout << "invalid!\n";
		else if (angle[0] == angle[1] && angle[0] == angle[2])
			cout << "equilateral\n";
		else if (angle[0] == angle[1] || angle[1] == angle[2])
			cout << "isosceles\n";
		else
			cout << "scalene\n";
	}
}
